### DISCUSSION TITLE
utilizar cartas en la prueba

### DISCUSSION THREAD

**Discussion by JaviBLM** [IIC2133-PUC/2024-1#131] [Ayuda] [answered]
Hola sé que probablemente no se pueda, pero no pierdo nada preguntando.
Si se me hace fácil seguir los algoritmos con cartas UNO, es posible usar estas en la i de mañana o lo único permitido sera el torpedo?

**Comment by Dafnemami** [comment, marked as answer]
Holis!, lleva una hoja en BLANCO o pídele una a los ayudantes, anótate allí unos números y con una tijera/regla te haces unas regias DYI cartas para que te vaya increíble en la prueba
